dev/io.openliberty.jakarta.rest.4.0.internal_fat_tck/publish/tckRunner/pom.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2024 IBM Corporation and others. All rights reserved.
+    This program and the accompanying materials are made available under the 
+    terms of the Eclipse Public License 2.0 which accompanies this distribution, 
+    and is available at 
+        http://www.eclipse.org/legal/epl-2.0/
+    
+    SPDX-License-Identifier: EPL-2.0 
+    Contributors: 
+        IBM Corporation - initial API and implementation
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.openliberty.jakarta.rest</groupId>
+    <artifactId>tck.runner</artifactId>
+    <version>4.0</version>
+    <packaging>pom</packaging>
+    <name>Jakarta REST 4.0 TCK Runner</name>
+
+    <properties>
+        <arquillian.version>1.7.0.Alpha13</arquillian.version>
```

Comment:
Mix of Alpha13 and Final between the two pom.xml files

Reply:
Please be a bit more specific?   What change are you suggesting?

Reply:
In this pom.xml it has 1.7.0.Alpha13 and the other one has 1.7.0.Final.  Change one so that they match.

Reply:
thanks